.ci/generate_bear_requirements.py

```diff
@@ -155,9 +156,11 @@ def write_npm_requirements(requirements):
 
     package_json_string = template.render(
         dependencies=npm_dependencies, version="0.8.0")
+    pretty_json = json.dumps(json.loads(package_json_string, object_pairs_hook=OrderedDict), indent=2)
```

Comment:
Something seems to be wrong with `codecov` config. Although other CI have 100% coverage, `codecov` shows only 82.11%.